### PR TITLE
API: fix p2p_conn struct naming

### PR DESCRIPTION
**Pull request by alex--m**

### CODE REVIEW

**Comment by manjugv** [comment]
@alex--m Looks good! Can you please add Huawei copyright ?

**Review by Sergei-Lebedev — APPROVED** [review]

**Review by vspetrov — APPROVED** [review]

**Comment by alex--m** [comment]
Seems to be an unrelated error:
````
--2021-01-07 14:31:52--  https://llvm.org/svn/llvm-project/cfe/trunk/tools/clang-format/git-clang-format
Resolving llvm.org (llvm.org)... 54.67.122.174
Connecting to llvm.org (llvm.org)|54.67.122.174|:443... connected.
HTTP request sent, awaiting response... 403 Forbidden
2021-01-07 14:31:52 ERROR 403: Forbidden.

Error: Process completed with exit code 8.
````

File: src/ucc/api/ucc.h [hunk 1]
```diff
@@ -2,6 +2,7 @@
  * @file ucc.h
  * @date 2020
  * @copyright Copyright (C) Mellanox Technologies Ltd. 2020.  ALL RIGHTS RESERVED.
+ * @copyright Copyright (C) Huawei Technologies Co., Ltd. 2020.  ALL RIGHTS RESERVED.
```
Comment (manjugv) [inline src/ucc/api/ucc.h:5]: Probably need to change the year from 2020 to 2021 :)

**Review by manjugv — COMMENTED** [review]
It seems like the clang issue is something with connection. Agree it is not an issue.

**Pull request merged**